src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -667,6 +668,40 @@ public override ImmutableArray<string> TupleElementNames
         }
 
         /// <summary>
+        /// Returns the symbols for all the tuple fields (in order and cached).
```

Comment:
Consider saying what base class is saying, i.e. "get the fields for the tuple's elements".